docs/SUMMARY.md

```diff
@@ -4,6 +4,7 @@
 - [Local Debug](./local_debug.md)
 - [AggOracle](./aggoracle.md)
 - [Aggsender](./aggsender.md)
+- [Aggsender Validator](./aggsender_validator.md)
```

Comment:
Can we make it a sub-chapter of the `Aggsender`, wdyt?

Reply:
Just a fuel for thought (doesn't need to be addressed now).

We should probably arrange the docs in the following way:
since the `aggsender` is a general, high level of abstraction, concept, IMO it probably makes sense to arrange the chapters this way:
- Aggsender
    - Aggsender Proposer
    - Aggsender Validator

WDYT?